src/listeners/ticket/close-modal-submit-validator.ts

```diff
@@ -0,0 +1,120 @@
+import { ApplyOptions } from '@sapphire/decorators';
+import { Events, Listener } from '@sapphire/framework';
+import {
+    ChannelType,
+    EmbedBuilder,
+    GuildMember,
+    type Interaction,
+    type ModalSubmitInteraction,
+} from 'discord.js';
+import { Color } from '../../lib/embeds';
+import { InputId, ModalId } from '../../lib/ticket';
+import { messageLogger } from '../..';
+import { userUrl } from '../../lib/user';
+import { ticketLogs } from '../..';
```

Comment:
The 2 imports from `'../..'` should be combined:
```suggestion
import { messageLogger, ticketLogs } from '../..';
```